cypress/support/commands.js

```diff
@@ -23,23 +23,28 @@ Cypress.Commands.add('autologin', () => {
 Cypress.Commands.add(
   'createContent',
   (contentType, contentId, contentTitle, path = '') => {
-    let api_url;
+    let api_url, auth;
```

Comment:
@ksuess I am wondering if we shouldn't use the autologin keyword to login before calling createContent. It crossed my mind as well to add auth to createContent directly. Though, we have to take into account that we will have different username/pw combinations in the future. I am a bit undecided on this...will think about it... :)

Reply:
TODO but not related to this issue: Reduce Cypress command "createContent" by authorization. A use of Cypress command autologin in a test should make 
      auth = {
        user: 'root',
        pass: 'root',
      };
in cy.request() superfluous. 
In this PR I just refactored, but did not change the Cypress command "createContent".
-> New Issue
-> Add task to Issue "Cypress best practice"

Reply:
I'd prefer if we would leave this out of this PR and handle that in a separate issue.

Reply:
I understand that it is not obvious. In this PR I just refactored, but **did not change** the Cypress command "createContent". Only difference to before: I added Guillotina login.